include/z64bgcheck.h

```diff
@@ -202,10 +202,16 @@ typedef struct {
     /* 0x60 */ f32 maxY;
 } BgActor; // size = 0x64
 
+#define BGACTOR_IN_USE (1 << 0) // The bgActor entry is in use
+#define BGACTOR_1 (1 << 1)
+#define BGACTOR_COLLISION_DISABLED (1 << 2) // The collision of the bgActor is disabled
+#define BGACTOR_CEILING_COLLISION_DISABLED (1 << 3) // The ceilings in the collision of the bgActor are ignored
+#define BGACTOR_FLOOR_COLLISION_DISABLED (1 << 5) // The ground in the collision of the bgActor are ignored
```

Comment:
```suggestion
#define BGACTOR_CEILING_COLLISION_DISABLED (1 << 3) // The ceiling collision of the bgActor is ignored
#define BGACTOR_FLOOR_COLLISION_DISABLED (1 << 5) // The floor collision of the bgActor is ignored
```
I haven't ported it over to MM yet, but "floor" and "ground" mean different things in bgcheck. Floors are specifically polys with a normal.y greater than 0.5f, whereas "ground" is a floor or wall poly with a normal.y >= 0 (see BgCheck_RaycastDownStaticList in OoT)